docker/README.md

```diff
@@ -34,7 +33,8 @@ For old version docker v1.10 below, Please replace ARG with ENV and set its valu
 Build from git repository
 
 ```bash
-  $ docker build -t NAME[:TAG] https://github.com/apache/incubator-mnemonic.git#:docker
+  $ docker build -t NAME[:TAG] https://github.com/apache/incubator-mnemonic.git#:docker:docker-centos
+  $ docker build -t NAME[:TAG] https://github.com/apache/incubator-mnemonic.git#:docker:docker-ubuntu
```

Comment:
Please note the use of letter case, using docker-Ubuntu instead of docker-ubuntu.
Please also correct the corresponding folder name